### PR TITLE
Project 16 - Astrid & Caroline 

### PR DESCRIPTION
**Pull request by CarolineF3**
Link to my deployed project: https://astrid-caroline-labyrinth.netlify.app/

### CODE REVIEW

**Review by Andrealarsson — APPROVED** [review]
Love the images and style! Great that you used styled-components. The reducer looks well planned and is not so long. Clean code that is easy to follow, and good naming of files and variables. 🦄 😃

File: code/src/components/EndScreen.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+import { useSelector } from "react-redux";
+import styled from "styled-components/macro";
+import img from "../assets/7.jpg";
```
Comment (KarolCislowski) [inline code/src/components/EndScreen.js:5]: remember to separate different types of input from each other with an empty line.

File: code/src/components/GameScreen.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+      ? img2
+      : props.coordinates === "1,1"
+      ? img3
+      : props.coordinates === "0,1"
+      ? img4
+      : props.coordinates === "0,2"
+      ? img5
+      : props.coordinates === "0,3"
+      ? img6
+      : "./assets/6.jpg"});
```
Comment (KarolCislowski) [inline code/src/components/GameScreen.js:73]: Generally, if you have long `if-else-if` or nested ternaries you should use `switch`.

In this scenario you could name your images for example `img-0,1` etc and then you just need to:
```jsx
  background-image: url(img-${props.coordinates})
```

**Review by KarolCislowski — APPROVED** [review]
Great app 🥳

**Pull request closed without merging**